src/mc/Y2calc.f03

```diff
@@ -0,0 +1,24 @@
+!---------------------------------------------------------------!
+!
+!     Written by Quinn MacPherson 8/4/17
+!---------------------------------------------------------------!
+subroutine Y2calc(u,Y2real)
+use params, only: dp 
+implicit none
+
+real(dp), intent(in) :: u(3)
+real(dp), intent(out) :: Y2real(5)
+real(dp), parameter, dimension(5) :: coefficient = (/1.092548430592079, &
```

Comment:
fixed as much as I can.  Replaced with locally defined PI.  Can't used pi from params as it isn't a parameter